src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/AnalyzerConfigOptionsExtensions.cs

```diff
@@ -55,7 +39,7 @@ private static T GetOptionWithAssertOnFailure<T>(AnalyzerConfigOptions analyzerC
             return value;
         }
 
-        public static bool TryGetEditorConfigOptionOrDefault<T>(this AnalyzerConfigOptions analyzerConfigOptions, TOption option, out T value)
+        private static bool TryGetEditorConfigOptionOrDefault<T>(this AnalyzerConfigOptions analyzerConfigOptions, TOption option, out T value)
             => TryGetEditorConfigOption(analyzerConfigOptions, option, (T?)option.DefaultValue, out value!);
 
         public static bool TryGetEditorConfigOption<T>(this AnalyzerConfigOptions analyzerConfigOptions, TOption option, [MaybeNullWhen(false)] out T value)
```

Comment:
@mavasani @tmat I'm confused by the implementation of this method. It's passing `default` literal to `defaultValue`. Why we're not always using `option.DefaultValue` (ie, `TryGetEditorConfigOptionOrDefault`)